### PR TITLE
Add websocket server with NSQd for nupad app

### PR DESCRIPTION
**Pull request by madhurjain2095**

### CODE REVIEW

**Comment by madhurjain2095** [comment]
The main advantage in using `websocketpp` is that it doesn't require any external dependency (like `Boost::Asio`, for example) and it runs entirely based on `C++11 STL` library

**Comment by sumeetgajjar** [comment]
Change Title

File: app/app_server.cpp [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+            changePublisherLoop_(),
+            publisher_(&changePublisherLoop_, evnsq::Option()),
+            publisherLoopThread_([this]() { changePublisherLoop_.Run(); }),
+            nsqdReady_(false) {
+        using websocketpp::lib::placeholders::_1;
+        using websocketpp::lib::placeholders::_2;
+        server_.init_asio();
+
+        // TODO: change bind to lambda
+        server_.set_open_handler(bind(&AppServer::onOpen, this,
```
Comment (sumeetgajjar) [inline app/app_server.cpp]: replace all binds with lambda

**Review by sumeetgajjar — APPROVED** [review]

File: app/app_server.cpp (hunk 1 above)
Comment (madhurjain2095) [reply app/app_server.cpp]: done

**Pull request merged**